.github/workflows/integration-tests.yml

```diff
@@ -132,6 +135,80 @@ jobs:
         with:
           triggered_by: ${{ env.TEST_TRIGGERED_BY }}
 
+  live-smoke-tests:
+    strategy:
+      matrix:
+        network: [GOERLI, OPTIMISM_GOERLI, ARBITRUM_GOERLI]
+    environment: integration
+    permissions:
+      checks: write
+      pull-requests: write
+      id-token: write
+      contents: read
+    name: ${{ matrix.network }} Smoke Tests
+    runs-on: ubuntu-latest
+    needs: [build-chainlink, changes]
+    env:
+      SELECTED_NETWORKS: ${{ matrix.network }}
+      CHAINLINK_COMMIT_SHA: ${{ github.sha }}
+      CHAINLINK_ENV_USER: ${{ github.actor }}
+      TEST_TRIGGERED_BY: core-CI-live
```

Comment:
They are all using the same triggered_by, if one finishes much earlier than another the cleanup will kill off the other 2 test runs in the matrix because it will cleanup all the namespaces. Something like this would work.

`TEST_TRIGGERED_BY: core-CI-live-${{ matrix.network }}`